src/Raven.Server/Documents/DatabasesLandlord.cs

```diff
@@ -355,6 +358,8 @@ public void DeleteDatabase(string dbName, DeletionInProgressStatus deletionInPro
                             _logger.Info("Could not create database configuration", ex);
                     }
 
+                    CheckDatabasePathsIntersection(dbName, configuration);
```

Comment:
There is a `if` statement bellow that says that configuration might be null. Shouldn't this be moved inside that `if` ?

Reply:
Done.